config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -538,6 +538,47 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-e2e-upgrade-1-21-1-22
+  - name: pull-cluster-api-provider-azure-e2e-apiversion-upgrade-v1beta1
```

Comment:
is this running both alpha3 -> beta1 and alpha4 -> beta1 ?

Reply:
```suggestion
  - name: pull-cluster-api-provider-azure-apiversion-upgrade
```

Reply:
yes